src/test/kotlin/calculator/OperatorTest.kt

```diff
@@ -0,0 +1,35 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class OperatorTest {
+    @Test()
+    fun `findBy 메서드는 전달받은 symbol과 일치하는 연산자 enum을 반환한다`() {
+        val expectedSymbol = "+"
+
+        val result = Operator.findBy(expectedSymbol)
+
+        assertThat(result.symbol).isEqualTo(expectedSymbol)
```

Comment:
```suggestion
        assertThat(result).isEqualTo(Operator.PLUS)
```
함수명에는 enum을 반환한다고 하였는데, 실제로는 enum 안의 value를 비교하고 있네요.
enum을 비교하도록 하면 어떨까요?